File: nodejs/app4/classexample/business_person.js
const { Person } = require("./iface/person");

module.exports = class BusinessPerson extends Person {
    constructor(name, age, country, job) {
        super(name, age, country);
        this.job = job
    }

    sayHello() {
        super.sayHello();
        console.log(`I'm work as ${this.job}.`);
    }
}

File: java/apps/app2/helloworldapps/src/main/java/example/cls/BusinessPerson.java
package example.cls;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class BusinessPerson implements Person {

    private String name;
    private int age;
    private String country;

    public BusinessPerson(String name, int age, String country) {
        this.name = name;
        this.age = age;
        this.country = country;
    }

    @Override
    public void sayHello() {
        System.out.println("My name is " + this.name + ".");
        System.out.println("I'm " + this.age + " years old.");
        System.out.println("I'm live in " + this.country + ".");
    }
    
}
